tests/PageCollector.test.ts

```diff
@@ -64,7 +64,11 @@ function getMockPage(): Page {
       return Promise.resolve(cdpSession as unknown as CDPSession);
     },
     ...mockListener(),
-  } as Page;
+    // @ts-expect-error internal API.
+    _client() {
```

Comment:
Probably we need to remove old createCDPSession

Reply:
will do in the follow-up CL.